ros_comm/ca23e58/ca23e58.bug

```diff
@@ -83,4 +83,4 @@ fault-codes:
   - CONCURRENCY:BAD-SYNC
 failure-codes:
   - SOFTWARE:BEHAVIORAL
```

Comment:
Behavioral seems off... But, then again, this is C++ and the issue falls under Underfined Behaviour, so I guess we can leave it as is.